starter_code/script.js

```diff
@@ -4,17 +4,44 @@ var aiPoint = 0;
 // This function returns the selection of the computer
 function getAISelection() {
     //TODO: randomly choose between 'rock', 'paper', or 'scissors'
+    var random = Math.floor(Math.random * 3);
+    var choices = ["Rock", "Paper", "Scissors"];
+    return choices[random];
 }
 
 // This function picks the winner
 function pickWinner(userValue, aiValue) {
     //TODO: pick the correct winner: user or ai
     //TODO: Add one point for the winner
+    if(userValue == aiValue){
+        return "draw";
+    } else if(userValue == "Rock"){
+        if(aiValue == "Paper"){
+            return "cpu";
+        } else {
+            return "user";
+        }
+    } else if(userValue == "Paper"){
+        if(aiValue == "Scissors"){
+            return "cpu";
+        } else {
+            return "user";
+        }
+    } else if(userValue == "Scissors"){
+        if(aiValue == "Rock"){
+            return "cpu";
+        } else {
+            return "user";
+        }
+    }
 }
 
 // This function sets the scoreboard with the correct points
 function setScore() {
-
+    var userText = $('#userPoint');
+    userText.text(userPoint);
+    var aiText = $('#aiPoint');
+    aiText.text(aiPoint);
```

Comment:
very nice! but just cause i have to be a stickler, why store the element in a var if we're only using it once? `$('#aiPoint').text(aiPoint)` is just as valid